database/database.go

```diff
@@ -48,6 +48,8 @@ var (
 	// dbConfiguration defines the name of the db table with configuration
 	// settings.
 	dbConfiguration = "configuration"
+	// dbAPIKeys defines the name of the db table with API keys for users.
+	dbAPIKeys = "api_keys"
```

Comment:
I've seen this a couple of times, I never really made a remark on it but I guess perhaps we should think to clean it up. In mongo you have a database that has collections, we often misuse the `db` prefix to label collections. This should be a collection I think so it should be `collAPIKeys`.

Another thing is the underscores, there's no real conventions around this but in general for collection names you want to go lowercase and avoid camel casing and underscores, I think a better collection name would be `apikeys`. Which is perfectly fine and is just way simpler.

Reply:
That's a good point about the prefix. I'll change that.

I'm not so sure about the collection name, though. We're already using a bunch of underscores around and changing that will need migration code. I'd rather keep the underscores.

Reply:
Agreed